### PR TITLE
First time player experience selection screen

### PR DESCRIPTION
**Pull request by MaxIsJoe**
part of #5316

image
image

CL: [New] New players will now receive a screen when joining for the first time to set their experience level.

### CODE REVIEW

**Comment by mmails** [comment]
You wrote "expierence" instead of "experience"

Under _Choose your experience level_

File: UnityProject/Assets/Scripts/UI/Systems/UIManager.cs [hunk 1]
```diff
@@ -85,6 +85,8 @@ public class UIManager : MonoBehaviour, IInitialise
 
 	public RoundEndScoreScreen ScoreScreen;
 
+	public ExpLevelUI FirstTimePlayerExpierenceScreen;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Systems/UIManager.cs]:
```suggestion
	public ExpLevelUI FirstTimePlayerExperienceScreen;
```

**Comment by corp-0** [comment]
ignore the magic numbers, but do encapsulate the fields as suggested by codacy. You know that is part of our coding guidelines.

**Comment by MaxIsJoe** [comment]
done

**Review by corp-0 — APPROVED** [review]

**Pull request merged**